web-marketplace/src/pages/CreateOrganization/CreateOrganization.utils.ts

```diff
@@ -30,8 +34,17 @@ export const hasTransferableProfile = (
   );
 };
 
-export const getCreateOrgSteps = (_: TranslatorType, hasProjects: boolean) =>
-  (
+export const getCreateOrgSteps = (_: TranslatorType, hasProjects: boolean) => {
+  let localStorageValue: FormData<OrganizationMultiStepData> | null = null;
+  try {
+    const value = localStorage.getItem(CREATE_ORGANIZATION_FORM_ID);
+    if (value) localStorageValue = JSON.parse(value);
+  } catch {}
+
+  const forceProjectsMigrationStep =
+    'selectedProjectIds' in (localStorageValue?.formValues ?? {});
```

Comment:
@alexander-astrand this is how I force the "migrate projects" step to be displayed even if current user has migrated all of his projects
if `selectedProjectIds` key is in the form values stored in local storage then that means the user has gone through this step, so we keep it for consistency
You can probably use the same pattern for "Personal info"